types/react-relay/index.d.ts

```diff
@@ -95,7 +95,7 @@ export interface QueryRendererProps<T extends RelayRuntimeTypes.OperationBase =
     dataFrom?: "NETWORK_ONLY"|"STORE_THEN_NETWORK";
     environment: RelayRuntimeTypes.Environment;
     query?: RelayRuntimeTypes.GraphQLTaggedNode | null;
-    render(readyState: ReadyState<T["response"]>): React.ReactElement | undefined | null;
+    render(readyState: ReadyState<T["response"]>): React.ReactNode;
```

Comment:
I can't give you a sign-off here. I have no idea whether this change is OK.

Reply:
Sorry, that commit got lumped into the PR. That commit is already on upstream. Only changes I made were to `svg4everybody`.

Thanks for responding to this so quickly.